### DISCUSSION TITLE
Add manufacturer name _TZ3210_jtifm80b to model TS0502B

### DISCUSSION THREAD

**Discussion by elle2** [Koenkk/zigbee2mqtt#11465] [Devices]
I've an issue with controller MiBoxer FUT035Z supported into zigbee2mqtt but not recognized.
The problem is that this device has the same model 'TS0502B' of FUT035Z but a different manufacturer name '_TZ3210_jtifm80b'.
Is it possible to add a new manufacturer name for the same device?
Thank you

PS. I use a docker version of zigbee2mqtt (release 1.23.0)

**Comment by Koenkk** [comment]
Added

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
